src/validators/object.ts

```diff
@@ -173,36 +176,60 @@ export abstract class ObjectValidator<T extends ObjectSchema = never, O = never>
           }
         }
 
+        // Serde
+        let serdeStr: string
+        let deriveMacro: string[] | undefined = undefined
+
+        if (this.deriveMacro !== undefined) {
```

Comment:
Non of the other options are rust specefic, why is this one needed, if I look at the examples it looks like deriveMacro is always the same expect for one case where Debug is not set, does it really matter if that is not set?

Reply:
Yes you want Eq, PartialEq or other stuff sometimes.

Reply:
To make matters worse, to be backwards compatible, we might need a custom field/macro above each field in an object/enum. #[serde(SOmething)] etc